src/ui/components/shared/log/AccountCardStatusIndicator.vue

```diff
@@ -11,12 +11,30 @@
   >
     mdi-check
   </v-icon>
-  <v-icon
+
+  <v-dialog
     v-else-if="accountStatus === accountStatusEnum.ERROR"
-    color="red"
+    width="500"
   >
-    mdi-alert
-  </v-icon>
+    <template v-slot:activator="{ on, attrs }">
+      <v-icon
+        v-bind="attrs"
+        color="red"
+        v-on="on"
+      >
+        mdi-alert
+      </v-icon>
+    </template>
+    <v-card>
+      <v-card-title class="headline red lighten-2">
+        Error
+      </v-card-title>
+      <v-card-text class="general-error-text subtitle-1">
+        {{ errorMessage }}
+      </v-card-text>
+    </v-card>
+  </v-dialog>
+
```

Comment:
When clicking on the Error icon, you will see the error message:

![Screenshot from 2021-02-02 22-30-47](https://user-images.githubusercontent.com/17686879/106659900-02c04c00-65a8-11eb-8dbb-1f1c3b9e0454.png)